storage/src/main/java/fk/prof/storage/AsyncStorage.java

```diff
@@ -15,7 +15,7 @@
      * @param path
      * @param content
      */
-    void storeAsync(String path, InputStream content);
+    void storeAsync(String path, InputStream content, long length);
```

Comment:
They usually do, yes. But here the method is used as fire and forget method, so the return value doesn't make that much sense. The implementation on the other hand can handle the result internally.